Madja16 Assignment 2

MemmoSJ opened the pull request:
Assignment 1

README.txt should provide all the information needed.

MemmoSJ committed:
initial commit

MemmoSJ committed:
exercise 1

MemmoSJ committed:
Merge branch 'master' into Madja16

MemmoSJ committed:
experimenting

MemmoSJ committed:
setup

nc-hela committed:
Exercises for week 2-5

nc-hela committed:
Create Week6Experiments.html

MemmoSJ committed:
assignment push

Done with assignment + folder cleanup

Kargo commented on Madja16/assignment_1/ImageFeed.php:
```diff
@@ -0,0 +1,105 @@
+<?php
+session_start();
+if (isset($_SESSION['session_id'])) {
```
Instead of wrapping the entire code in an if/else, you can use exit() to stop execution after the initial check:
`if (!isset($_SESSION['session_id'])) {
    header("Location: index.php?error=nologindetected");
    exit();
}`

You're already doing it further down.

Kargo reviewed (approved):
Looks good 👍

nc-hela committed:
mvc exercise

nc-hela committed:
Update router for case sensitivity

nc-hela committed:
Made an api that exposes a list of users with no security

nc-hela committed:
Create Week7Apis.php

nc-hela committed:
Create Week7OwnApi.php

nc-hela committed:
Create Week9Bootstrap.html

nc-hela committed:
Week9Exercises added

nc-hela committed:
Bootstrap exercises

MemmoSJ committed:
Merge remote-tracking branch 'origin/Exercises' into Madja16

MemmoSJ committed:
gotta redo a lot because of dumb json format

MemmoSJ committed:
assignment 2

MemmoSJ committed:
ignore note files

MemmoSJ committed:
small video 

shows home index
shows sign up form (and ajax call)
shows login form
shows test.php results
shows user list
shows upload page (and example of xss)
shows image feed
